### PR TITLE
Task2

### PR DESCRIPTION
**Pull request by SimonasVsm**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
+const previousImage = document.getElementById('left-btn')
+const nextImage = document.getElementById('right-btn')
+const carouselImages = document.querySelectorAll('.carousel__image')
+const carouselIndicatorButtons = document.querySelectorAll(
+	'.carousel__indicator'
+)
```
Comment (visma-zanas-stundys) [inline script.js]: Consider using a namespace for ui elements ;)

```js
const ui = {
  buttonNext: document.getElementById('...'),
  buttonPrevious document.getElementById('...'),
  get currentImageElement() {
    return document.querySelectorAll('.carousel__image')[currentImage];
  }
}
```

File: script.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+		addActiveNavButtonClass(currentImage)
+		currentImage--
+	}
+})
+
+nextImage.addEventListener('click', () => {
+	removeActiveImageClass()
+
+	if (currentImage + 1 >= carouselImages.length + 1) {
```
Comment (visma-zanas-stundys) [inline script.js]: This condition is a bit hard to read. Try extracting to a variable.

```
const isLast = currentImage + 1 >= carouselImages.length + 1;

if (isLast) {
  // ...
} else { ...}

File: script.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+		addActiveImageClass(carouselImages[currentImage - 1])
+
+		currentImage++
+		addActiveNavButtonClass(currentImage)
+	}
+})
+
+carouselIndicatorButtons.forEach((button) => {
+	button.addEventListener('click', () => {
+		const imageSelected = carouselImages[button.id - 1]
```
Comment (visma-zanas-stundys) [inline script.js]: I see some playing around with indices. Consider simplifying a bit ;) Don't take long ;) Up to 30 mins.

File: styles.css [hunk 4]
```diff
@@ -1,179 +1,293 @@
...
+	text-decoration: none;
+}
+
+a:active,
+a:hover {
+	color: var(--activeBtn);
+}
+
+.sticky {
+	position: fixed;
```
Comment (LukasTyla) [inline styles.css]: Do note that there is also newer, albeit less widely supported option of `sticky` 😉

**Review by LukasTyla — COMMENTED** [review]
Looks good overall.

**Pull request merged**